assets/js/components/consent-mode/WPConsentAPIRequirements.js

```diff
@@ -59,6 +60,14 @@ export default function WPConsentAPIRequirements() {
 	const { installActivateWPConsentAPI, activateConsentAPI } =
 		useDispatch( CORE_SITE );
 
+	// Check for `getNonces` errors in the user store, mostly to cover
+	// the case if user is offline, but also any other potential
+	// case that nonces fetch fails. As `installActivateWPConsentAPI`
+	// action will invoke fetch for nonce, and when offline this will fail.
+	const fetNoncesAPIError = useSelect( ( select ) =>
+		select( CORE_USER ).getErrorForAction( 'getNonces' )
+	);
```

Comment:
Thanks, @zutigrm, but I think we should do a better job here. The action that we call and want to receive errors for is `installActivateWPConsentAPI`, not `getNonces` which is actually a selector, not an action. So we need to update `installActivateWPConsentAPI` to properly set the error state if something doesn't work correctly.

In particular, if we can't get nonces (`updates` nonce returns `undefined`), we need to set the error manually (see create-error-store.js) and not call the `fetchInstallActivateWPConsentAPI` action at all. Then we need to update this section to get an error for the  `installActivateWPConsentAPI` action specifically.